### PR TITLE
Revert "Remove `hello` multi-arch image build"

### PR DESCRIPTION
**Pull request by edsantiago**
This reverts commit 7e0130f75cc27ae32d8843a8be8d2588aa2dcee9.

It broke CI, not just on podman but on Buildah too. Buildah bud
tests require the hello subdirectory.
```release-note
None
```

### CODE REVIEW

**Review by vrothberg — APPROVED** [review]
LGTM

**Comment by rhatdan** [comment]
I think the best fix for this is to stop buildah from relying on podman for its tests.

https://github.com/containers/buildah/pull/4994

**Comment by edsantiago** [comment]
>stop buildah from relying on podman for its tests

Agreed but that then needs to propagate to all active buildah branches, *and* all active podman branches which run buildah-bud tests. That will take days of dedicated effort. This PR is still necessary.

**Comment by edsantiago** [comment]
CI is broken. This is an emergency to get PRs flowing. I would like to merge this right now (force-merge, since, duh, CI is broken). We can nitpick later.

@containers/podman-maintainers mergeme

**Comment by vrothberg** [comment]
@rhatdan can you force merge?  It will turn green after this PR merges.

**Comment by cevich** [comment]
I can :smile: 

> CI is broken. This is an emergency to get PRs flowing. I would like to merge this right now (force-merge, since, duh, CI is broken). We can nitpick later.

That's fine, just thinking to getting a head start on the list.

**Comment by edsantiago** [comment]
Thank you @cevich!

@vrothberg @rhatdan @Luap99 @jakecorrenti @mtrmac @n1hility please rebase and repush

**Pull request merged**
